spec/models/entry_filter_spec.rb

```diff
@@ -7,20 +7,32 @@
 
   it "#clients" do
     filter = EntryFilter.new
-    expect(Client).to receive(:by_name)
-    filter.clients
+
+    client1 = create(:client, name: "Zack")
+    client2 = create(:client, name: "Andy")
+    clients = [client2, client1]
```

Comment:
Maybe use `create_list` here?

Reply:
Sure, I was originally also testing the ordering here, but I guess you could argue that's handled by tests on `Client` anyway :+1: